examples/local/cert-operator-lab-chart/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+name: cert-operator-lab-chart
+version: 0.1.0
```

Comment:
Why exactly this version? I guess it's just random for the local stuff?

Reply:
yep, just a semver to make helm happy

Reply:
Ack I personally prefer `1.0.0` for stuff like that, but no need to change.